src/web3/evm/connector/gnosisSafe/triggers/execution.ts

```diff
@@ -1,24 +1,26 @@
 import axios from "axios";
-import { ConnectorInput, TriggerBase } from "grindery-nexus-common-utils";
+import { ConnectorInput, TriggerBase, TriggerInit } from "grindery-nexus-common-utils";
 import { sanitizeParameters } from "../../../../../utils";
 import * as evm from "../../../triggers";
 import { API_BASE, processSafeTxInfo } from "../common";
+import { NewEventInput, TriggerBasePayload, TriggerBaseState } from "../../../../utils";
 
 export async function safeTransactionExecuted(
   input: ConnectorInput,
   predicate = (_payload: Record<string, unknown>) => true
 ): Promise<TriggerBase> {
   const ret = new evm.NewEventTrigger(
-    await sanitizeParameters({
+    (await sanitizeParameters({
       ...input,
       fields: {
         ...(input.fields as object),
         eventDeclaration: "ExecutionSuccess(bytes32 txHash, uint256 payment)",
         parameterFilters: {},
       },
-    })
+    })) as TriggerInit<NewEventInput, TriggerBasePayload, TriggerBaseState>
```

Comment:
We should not add casting here, instead we should change type of `input` to `TriggerInit`. This is a recent change in the common library, trigger factory function accepts `TriggerInit` instead of `ConnectorInput` now. `sanitizeParameters` may need update as well.

Reply:
@SAPikachu Yes I observed that, but I had a doubt because the `sanitizeParameters` function is used in other places of the code and also for several actions in order to reorder some input parameters. In that case, it's a bit strange to type the input as a `TriggerInit` if it's an action, isn't it?